SubEthaEdit-Mac/Source/MultiPagePrintView.h

```diff
@@ -15,7 +15,7 @@
     NSSize I_textContainerSize;
     NSPoint I_textContainerOrigin;
     PlainTextDocument *I_document;
-    __weak NSTextView *I_headerTextView;
+    NSTextView *I_headerTextView;
```

Comment:
The var only was "assign" as it was retained bey the view hierarchy. No Problem in making it strong here.